tests/src/test_cases/test_mem.c

```diff
@@ -0,0 +1,40 @@
+#if LV_BUILD_TEST
+#include "../lvgl.h"
+
+#include "unity/unity.h"
+
+void setUp(void)
+{
+    /* Function run before every test */
+}
+
+void tearDown(void)
+{
+    /* Function run after every test */
+}
+
+/* Non-zero size with null pointer should behave like malloc */
+void test_realloc_safe_behave_malloc_like(void)
+{
+    void * ptr = NULL;
```

Comment:
You need only one variable, either ptr or data, but not both.

Reply:
Thanks, fixed in 986a62d